spec/scenic/definition_spec.rb

```diff
@@ -2,22 +2,51 @@
 
 module Scenic
   describe Definition do
+    describe "find_definition" do
```

Comment:
Should this be a tested public method? Or should it be private and tested as part of other tests in the app?